bundles/org.openhab.binding.bluelink/README.md

```diff
@@ -0,0 +1,222 @@
+# Bluelink Binding (Hyundai/Genesis)
+
+This binding integrates Hyundai vehicles equipped with Bluelink connected car services.
+It allows you to monitor your vehicle's status and control various features remotely.
+
+Genesis vehicles may also work, though this has not been tested.
+
+**NOTE: The binding only support the US region at the moment, because the Bluelink API
+differs by region.**
+
+## Supported Things
+
+| Thing Type | Description                                  |
+|------------|----------------------------------------------|
+| `account`  | Bridge representing your Bluelink account    |
+| `vehicle`  | A vehicle connected to your Bluelink account |
+
+## Discovery
+
+Once an account bridge is configured and online, the binding will automatically discover vehicles
+registered to your account.
+
+## Actions
+
+Vehicle things support the following actions:
+
+* Force Refresh: Requests up-to-date data from the vehicle and updates the channels.
+* Climate Start
+* Climate Stop
+
+## Thing Configuration
+
+### Account Bridge
+
+| Parameter  | Required | Description                                              |
+|------------|----------|----------------------------------------------------------|
+| `username` | Yes      | Bluelink account email                                   |
+| `password` | Yes      | Bluelink account password                                |
+| `pin`      | No       | Bluelink service PIN (required for lock/unlock commands) |
```

Comment:
Might be way to early, but ask myself if it would be usefull to already add a region parameter to allow different authentication/endpoint paths in the code to sort of make it forward compatible when other regions are added.

Reply:
Thanks for the detailed review! I have done this on a [branch](https://github.com/marcusb/openhab-addons/tree/canada) that adds support for CA (but am holding off on making a PR until I can get someone to test it). I will backport the region selection part.